### PR TITLE
Drcandacemakedamoore/updating

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
Added an environment yaml to be tried, folder per episode structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +12,28 @@ Image basics (0:30 hours)
 Working with skimage (0:30 hours)
 Drawing and Bitwise Operations (2:30 hours)
 Creating Histograms (1:00 hours)
-Blurring Images (1:30 hours)
+Summary lecture(1:00 hours)
+
+Day 2:
+Blurring Images (1:00 hours)
 Thresholding (1:00 hours)
 Connected Components Analysis (1:00 hours)
```
Comment (bvreede) [inline README.md:20]:
```suggestion
Day 2:
- Blurring Images (1:00 hours)
- Thresholding (1:00 hours)
- Connected Components Analysis (1:00 hours)
```

**Review by bvreede — APPROVED** [review]
Nice!
I didn't check whether the environment works, but all looks good! Some small comments below.

File: README.md [hunk 2]
```diff
@@ -13,15 +12,28 @@ Image basics (0:30 hours)
...
 
 The course will also be taught [interally](internal_workshop.md).
 
+## Environment:
+    To work with the correct packages:
+    Install all Python packages required, using conda and the
+    `environment.yml` file.
+
+    * The command for Windows/Anaconda users will be:
+     `conda env create -f environment.yml`.
```
Comment (bvreede) [inline README.md:36]:
```suggestion
To work with the correct packages:
Install all Python packages required, using conda and the`environment.yml` file.

The command for Windows/Anaconda users will be:
```
conda env create -f environment.yml
```
```

Comment (bvreede) [reply README.md:36]: I don't think this suggestion works :/ sorry.
Here, I would take all text out of the code block, and only include the `conda env create...` command in a code block.

File: README.md [hunk 3]
```diff
@@ -13,15 +12,28 @@ Image basics (0:30 hours)
 Working with skimage (0:30 hours)
 Drawing and Bitwise Operations (2:30 hours)
 Creating Histograms (1:00 hours)
-Blurring Images (1:30 hours)
+Summary lecture(1:00 hours)
+
+Day 2:
```
[the remark this answers is not in the document]
Comment (drcandacemakedamoore) [reply README.md:17]: OK, to be fixed later

**Pull request merged**